templates/nodejs/eslintignore.tpl

```diff
@@ -1,2 +1,3 @@
 node_modules/*
 samples/node_modules/*
+doc_*.js
```

Comment:
Note, I think these file names are on the verge of changing, and I don't think this regex will match correctly.